easyrest/models/restaurant.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    Text,
+)
+
+from .meta import Base
+
+
+class Restaurant(Base):
+    """
+    The data model of Restaurant table
+    Defines data stricture of restaurant table
+    """
+    __tablename__ = 'Restaurant'
```

Comment:
As I already mentioned in #3 it's very common to use snake case fro table names and use plural nouns as table names.